File: app/backend/src/main/java/eaproject/input/CreateTypeInput.java
package eaproject.input;

import java.io.Serializable;
import java.sql.Timestamp;

public class CreateTypeInput extends BaseInput implements Serializable {
    private String name;
    private Integer numberOfPersons;
    private Timestamp creationDate;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getNumberOfPersons() {
        return numberOfPersons;
    }

    public void setNumberOfPersons(Integer numberOfPersons) {
        this.numberOfPersons = numberOfPersons;
    }

    public Timestamp getCreationDate() {
        return creationDate;
    }

    public void setCreationDate(Timestamp creationDate) {
        this.creationDate = creationDate;
    }
}

File: app/frontend/ea-project/src/models/input/CreateTypeInput.js
import BaseInput from "./BaseInput";

export default class CreateTypeInput extends BaseInput {
  constructor(name = '', numberOfPersons = 0, creationDate = new Date(), lazyLoad = false) {
    super(lazyLoad);
    this.name = name;
    this.numberOfPersons = numberOfPersons;
    this.creationDate = creationDate;
  }
}